app/models/users/school_management.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Users
+  class SchoolManagement < User
+    include UserAdmin
+
+    validates :first_name,
+              :last_name,
+              :role,
+              presence: true
+
+    validates :email, format: /\A[^@\s]+@ac-[^@\s]+\z/, if: :school_manager?
```

Comment:
TODO/DOC: missing idea that school manager mails in ac-xxx.xx ensure proper rights on other school_management registration